tsconfig.json

```diff
@@ -17,7 +17,5 @@
     "types": ["jest", "node", "react", "styled-components-react-native"],
     "skipLibCheck": true
   },
-  "exclude": [
-    "node_modules", "babel.config.js", "metro.config.js", "jest.config.js", "tools"
-  ]
+  "include": ["src/**/*", "./jest.setup.ts", "index.js", "jest.config.ts", "metro.config.ci.js", "metro.config.js"]
```

Comment:
This e.g. excludes `App.tsx`, right? Why not exclude like we did before?

Reply:
No, `**` matches zero or more directories. Changed it so it is more similar to integreat.